e2e/install/install_test.go

```diff
@@ -0,0 +1,138 @@
+package basic_test
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"path"
+	"runtime"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/deprecated/scheme"
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var cfg *rest.Config
+var k8sClient client.Client
+var k8sManager ctrl.Manager
+var testEnv *envtest.Environment
+
+// Helper for passing yaml to kubectl
+func applyYaml(rootDir string, filename string) {
+	cmd := exec.Command("kubectl", "apply", "-f", filename)
+	cmd.Dir = rootDir
+	_, err := cmd.CombinedOutput()
+	Expect(err).ToNot(HaveOccurred())
+}
+
+// Encapsulate the logic to find where the deploy yamls are
+func getDeployYamlDir() string {
+	_, filename, _, _ := runtime.Caller(0)
+	return path.Clean(filename + "/../../../deploy")
+}
+
+// Encapsulate the logic to find where the csi crd yamls are
+func getCsiYamlDir() string {
+	_, filename, _, _ := runtime.Caller(0)
+	return path.Clean(filename + "/../../../csi/moac/crds")
+}
+
+// We expect this to fail a few times before it succeeds,
+// so no throwing errors from here.
+func mayastorReadyPodCount() int {
+	var mayastorDaemonSet appsv1.DaemonSet
+	if k8sClient.Get(context.TODO(), types.NamespacedName{Name: "mayastor", Namespace: "mayastor"}, &mayastorDaemonSet) != nil {
+		fmt.Println("Failed to get mayastor DaemonSet")
+		return -1
+	}
+
+	return int(mayastorDaemonSet.Status.CurrentNumberScheduled)
+}
+
+// Install mayastor on the cluster under test.
+// We deliberately call out to kubectl, rather than constructing the client-go
+// objects, so that we can verfiy the local deploy yamls are correct.
+func installMayastor() {
+	deployDir := getDeployYamlDir()
+	applyYaml(deployDir, "namespace.yaml")
+	applyYaml(deployDir, "moac-rbac.yaml")
+	applyYaml(getCsiYamlDir(), "mayastorpool.yaml")
```

Comment:
```
$ ls -l deploy/mayastorpoolcrd.yaml 
lrwxrwxrwx 1 xxxx xxxx 34 Sep  8 13:51 deploy/mayastorpoolcrd.yaml -> ../csi/moac/crds/mayastorpool.yaml
```
May be simpler to just use files only from the deploy directory.

Reply:
Oh, didn't see that. Done.